lib/client/cluster_client.go

```diff
@@ -690,14 +763,17 @@ func PerformSessionMFACeremony(ctx context.Context, params PerformSessionMFACere
 	log.DebugContext(ctx, "Issuing single-use certificate from unary GenerateUserCerts")
 	newCerts, err := rootClient.GenerateUserCerts(ctx, *certsReq)
 	if err != nil {
-		return nil, nil, trace.Wrap(err)
+		return nil, trace.Wrap(err)
 	}
 
 	keyRing := params.KeyRing
 
 	// Nothing more to do.
 	if keyRing == nil {
-		return nil, newCerts, nil
+		return &PerformSessionMFACeremonyResult{
+			NewCerts:    newCerts,
+			MFAResponse: mfaResp,
```

Comment:
I think I'm missing something - why do we still set `MFAResponse: mfaResp` when `allowReuse == false`? Will this be treated like an "expired" mfa response by the caller?

Reply:
[637ed2d](https://github.com/gravitational/teleport/pull/54069/commits/637ed2d5069c8db0851e91d27cec8ab144d0dc65)